network-api/networkapi/wagtailpages/utils.py

```diff
@@ -469,3 +469,10 @@ def get_default_locale():
 def get_original_by_slug(Model, slug):
     (DEFAULT_LOCALE, DEFAULT_LOCALE_ID) = get_default_locale()
     return Model.objects.get(slug=slug, locale=DEFAULT_LOCALE_ID)
+
+
+def requireShareFields(form):
```

Comment:
@danielfmiranda What do you think about changing this to `snake_case` which is more common in Python?

Reply:
good catch Tibor, updated!